newdle/client/src/components/Footer.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import {Grid} from 'semantic-ui-react';
+import {Trans} from '@lingui/macro';
+import LanguageSelector from './common/LanguageSelector';
+import styles from './Footer.module.scss';
+
+export default function Footer() {
+  const FOOTER_LINKS = {
+    'Help': '#',
+    'Contact': '#',
+    'Terms and conditions': '#',
+    'Another link': '#',
+  };
+
+  return (
+    <footer className={styles.footer}>
+      <Grid columns={3}>
+        <Grid.Row verticalAlign="middle">
+          <Grid.Column computer={4} tablet={4} only="computer tablet">
+            <ProjectInfo />
+          </Grid.Column>
+          <Grid.Column mobile={16} tablet={8} computer={8}>
+            <p className={styles.links}>
+              {Object.keys(FOOTER_LINKS).map((key, pos, arr) => (
+                <React.Fragment key={key}>
+                  <a
+                    href={FOOTER_LINKS[key]}
+                    className={styles.link}
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    {key}
+                  </a>
+                  {pos < arr.length - 1 ? <span className={styles.separator}>{' | '}</span> : null}
```

Comment:
an alternative to this could be a `<ul>` that's styled as an inline list with `content: ' | '` in the elements' `::before` (using a `li + li` selector to avoid it on the first element)

Reply:
I just did this, pretty neat trick! thanks for the suggestion